### PR TITLE
Feature/mailservice

### PR DESCRIPTION
**Pull request by Aswanth-c**
added mail configuration by SendGrid 
in order to make  it work you need to add  SENDGRID_API_KEY and MAIL_FROM in env local 
and also make sure MAIL_FROM mail-id  is verified on SendGrid

### CODE REVIEW

File: mail/template/applicationreceived.handlebars [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>Hey {{ name }},</p>
+<p>your application has been  received </p>
+<p>
```
Comment (tomknig) [inline mail/template/applicationreceived.handlebars:3]: There are some extra spaces and a missing closing tag here.
```suggestion
<p>your application has been received.</p>
<p></p> /* or remove this tag altogether */
```

**Review by tomknig — CHANGES_REQUESTED** [review]
1. There are some inconsistencies here wrt. indentation. CI is passing because the new directory `mail` is not yet included for linting. We should add it in the `package.json`.
2. Is this really the approach we want to take? I think manually sent mails, without proper dkim detup etc., are quite likely to bounce. How about using email SaaS for that?

File: mail/template/applicationreceived.handlebars (hunk 1 above)
Comment (kurdybacha) [reply mail/template/applicationreceived.handlebars:3]: I don't have much experience with this approach. Does it require some smtp server configuration? If yes then maybe the simplest and most reliable would be to use some SaaS service as @tomknig is suggesting. What do you think @Aswanth-c ?

Comment (Aswanth-c) [reply mail/template/applicationreceived.handlebars:3]: yes it required smtp connection.  im not sure which way is good. if using a saas is better than this we can go forward with it

**Review by tomknig — CHANGES_REQUESTED** [review]
I guess this is a good starting point. Thx!
Few observations:
1. Please remove the unused dependencies `nodemailer`, `nodemailer-express-handlebars`.
2. Please add the required dependency `@sendgrid/mail`
3. Please add `mail` to the `lint` and `format` scripts in the `package.json`. I see some inconsistent formatting.

**Pull request merged**